src/functionalTest/resources/features/F-004 - Post recordSittingRecords endpoint/F-004.feature

```diff
@@ -536,4 +536,24 @@ Feature: F-004 - Scenarios for the POST /recordSittingRecords endpoint
     And the request body contains the "payload with one sitting record" as in "F-004_allFields"
     And a call is submitted to the "RecordSittingRecords" endpoint using a "POST" request
     Then a "negative" response is received with a "400 Bad Request" status code
-    And the response contains "errors[0].message" as "004 unknown hmctsServiceCode"
\ No newline at end of file
+    And the response contains "errors[0].message" as "004 unknown hmctsServiceCode"
+
+  @S-004.38 @PossibleDuplicates #IJPS-124
+  Scenario: Negative response, check whether correct record is returned - potential duplicate
+    Given a user with the IDAM role of "jps-recorder"
+    And "one" record for the hmctsServiceCode "BBA3" exists in the database with the payload "S-004.38_Existing_1" 
+    And "a second" record for the hmctsServiceCode "BBA3" exists in the database with the payload "S-004.38_Existing_2" 
+    When a request is prepared with appropriate values
+    And the request contains a valid service token
+    And the request contains the "hmctsServiceCode" as "BBA3"
+    And the request body contains the "sittingDate, epimmsId, personalCode matching the existing record, but AM/PM does not match" as in "S-004.38"
```

Comment:
shouldn't it be judgeRoleTypeId that doesn't match, and AM/PM match? Also would be good to say which record (ie. matching the existing record 1.)